lib/src/lib.rs

```diff
@@ -803,7 +928,8 @@ mod test_authz {
         } else {
             id_cred_i.get_ccs().unwrap()
         };
-        let (mut _responder, r_prk_out) = responder.verify_message_3(valid_cred_i).unwrap();
+        let (mut _responder, r_prk_out, r_prk_exporter) =
+            responder.verify_message_3(valid_cred_i).unwrap();
```

Comment:
As discussed offline, modify to something like this:

```rust
        let (mut _responder, r_prk_out, r_prk_exporter) =
            responder.verify_message_3(valid_cred_i).unwrap().complete_without_message_4();
```

which:
1. is more explicit
2. makes fields related to message_3 (`prk_4e3m`, `th_4`) be inaccessible after this point